x-pack/plugins/security_solution/public/common/components/grouping/empty_resuls_panel.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiImage, EuiPanel, EuiText, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import type { CoreStart } from '@kbn/core/public';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+
+const panelStyle = {
```

Comment:
Why the fixed width and height values? If you can add a comment, that'd be awesome!

Reply:
I just reused the existing empty panel code - we will work on the better design later and change. So for MVP can keep it like this.